src/core/Alert/Alert/Alert.baseStyles.ts

```diff
@@ -42,14 +49,13 @@ export const baseStyles = (theme: SuomifiTheme) => css`
       width: 14px;
       height: 14px;
       margin-left: ${theme.spacing.xxs};
-      margin-top: ${theme.spacing.xxs};
-      transform: translateY(1px);
+      transform: translateY(0.1em);
     }
   }
 
   /** Small screen variant styles */
-  &--small-screen {
-    & .fi-alert-close-button-wrapper {
+  &.fi-alert--small-screen {
```

Comment:
Because the styles were broken otherwise. After the separation to two components the selector is no longer directly adjacent to the `.fi-alert` class, and thus now requires that beginning to be added.